src/Halogen/Query/HalogenM.purs

```diff
@@ -179,6 +187,10 @@ derive newtype instance ordForkId :: Ord ForkId
 -- | Some care needs to be taken when using a `fork` that can modify the
 -- | component state, as it's easy for the forked process to "clobber" the state
 -- | (overwrite some or all of it with an old value) by mistake.
+-- |
+-- | When a component is disposed of any active forks will automatically
+-- | be killed. New forks can be started during finalization but there will be
+-- | no means of killing them.
```

Comment:
So during handling of `finalizer` action if one forks something it will not be killed, i.e. only computations forked before finalize action are killed.

Is that correct? would be nice to have a test of this behavior.